udata/frontend/__init__.py

```diff
@@ -17,6 +19,50 @@
 
 _template_hooks = {}
 
+
+@hook.app_template_global()
+def manifest(app, filename, **kwargs):
```

Comment:
Could we do without manifest, in_manifest and jsonld filters in udata, to prevent code duplication?